lib/SPIRV/SPIRVReader.cpp

```diff
@@ -3527,6 +3528,35 @@ void SPIRVToLLVM::transIntelFPGADecorations(SPIRVValue *BV, Value *V) {
   }
 }
 
+void SPIRVToLLVM::transUserSemantic(unsigned FunNum) {
```

Comment:
I suggest to accept `Function *` here instead of index of function in the module